rust/pattern/constraint/thing_constraint.rs

```diff
@@ -64,9 +67,9 @@ impl TryFrom<String> for IIDConstraint {
     }
 }
 
-impl Display for IIDConstraint {
+impl fmt::Display for IIDConstraint {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "iid {}", self.iid)
+        write!(f, "{} {}", Iid, self.iid)
```

Comment:
Many hardcoded keywords are replaced with elements of string enums from common/token.